test/spec/features/popup-menu/ReplaceMenuProviderSpec.js

```diff
@@ -658,8 +667,8 @@ describe('features/popup-menu - replace menu provider', function() {
           openPopup(startEvent);
 
           // then
-          expect(queryEntry('replace-with-none-start')).to.be.null;
-          expect(queryEntries()).to.have.length(6);
+          expect(queryEntry('replace-with-none-start'), container).to.be.null;
```

Comment:
Wrong position of container argument.

Reply:
We can get rid of the container argument entirely:

```javascript
function queryEntry(selector) {
  const container = getBpmnJS().get('canvas')._container;

  ...
}
```

Reply:
Fixed.